src/Nethermind/Nethermind.BeaconNode.Host/Nethermind.BeaconNode.Host.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Essential.LoggerProvider.Elasticsearch" Version="1.1.1" />
+    <PackageReference Include="Essential.LoggerProvider.Elasticsearch" Version="1.3.2" />
```

Comment:
what do we use elasticsearch for?
we use seq, grafana and prometheus for eth1, can we reuse that stack?

Reply:
Easy enough to change to Seq (dependency only exists in the host). Just getting to the point where I need something to scan the logs beyond just scrolling back in the console :-)

One point in favour of ES is that it is open source, compared to Seq (although Seq does have single user free licence level).

Re: Prometheus, haven't got to the point of adding counters, etc yet, although the Eth2 spec does include a definition of what metrics are needed (when we get up to it).